code/modules/projectiles/projectile.dm

```diff
@@ -606,6 +606,15 @@
 		if((L.mobility_flags & MOBILITY_USE|MOBILITY_STAND|MOBILITY_MOVE) & (MOBILITY_USE|MOBILITY_MOVE))	//If they're either able to move or use items, while crawling, they're not stunned enough to avoid projectiles.
 			Bump(AM)
 
+/obj/item/projectile/Move(atom/newloc, dir = NONE)
+	. = ..()
+	if(.)
+		if(temporary_unstoppable_movement)
+			temporary_unstoppable_movement = FALSE
+			DISABLE_BITFIELD(movement_type, UNSTOPPABLE)
+		if(fired && can_hit_target(original, permutated, TRUE))
+			Bump(original))

```

Comment:
nich

you're bad at parentheses. 

```suggestion
			Bump(original)
```

Reply:
This isn't my PR, it's a mirror :)

Reply:
yes but the original PR did not have this error.